src/main/java/nbgame/engine/PathDriver.java

```diff
@@ -0,0 +1,22 @@
+package nbgame.engine;
+
+import nbgame.game.Statistic;
+
+import java.util.List;
+
+public class PathDriver {
+
+    public String getPicPath(String imagePath) {
+        String path = "";
+
+        try {
+            ClassLoader loader = getClass().getClassLoader();
+            path = loader.getResource(imagePath).toString();
+            path = path.replaceAll("%20", " ");
+        } catch (Exception ex) {
+            System.out.println(ex.getMessage());
```

Comment:
Tutaj wyciszasz wyjątek. Czemu ma to służyć?
Zamiast System.out.println dodaj do projektu slf4j i użyj logerów. https://www.baeldung.com/slf4j-with-log4j2-logback